torch/_inductor/codegen/common.py

```diff
@@ -2024,7 +2025,24 @@ def do_cse(v):
                             bounds=bounds,
                             dtype=output_dtype,
                         )
+
+                        nonlocal idx
```

Comment:
```suggestion
                        nonlocal output_idx
```

Suggest a more informative name